client/packages/common/src/intl/strings/IntlStrings.ts

```diff
@@ -18,7 +19,8 @@ export interface TypedTFunction<Keys> {
 //
 // returned function can be used with optional ns, i.e. t('label.create-user', { ns: 'system' })
 export const useTranslation = (ns?: Namespace): TypedTFunction<LocaleKey> => {
-  const { t } = useTranslationNext(ns);
+  const {i18n} = useIntl();
+  const { t } = useTranslationNext(ns, {i18n});
```

Comment:
Why was this changed?

Reply:
in order to share the i18n object between the host app and the plugins - to do so, I've had to keep the `i18n` in a shared context, and here retrieve it and us that as an option when calling `useTranslation`. The original problem is that the host app is able to translate, and knows the current language etc - but the plugins are operating in a different context and they are unable to see the current intl configuration.